dbms/tests/queries/0_stateless/00607_materialized_view_partition.reference

```diff
@@ -0,0 +1,17 @@
+## Add 2 partitions
+201802
+201803
+## Drop partition 201802
+201803
+## Drop partition 201803
+## Add 3 partitions
+201801
+201802
+201803
+## Detach partition 201803
```

Comment:
Maybe use `-` is better? `#` looks like a comment

Reply:
I think it would be better to use one style for this, but so far there is no uniform style )